hbase/src/main/resources/hbase/bin/hirb.rb

```diff
@@ -0,0 +1,189 @@
+#
```

Comment:
Yes. This is. I've added it for manual testing only. Refer to `hbase.irb.load` in the javadoc. I can remove it. It mainly helps me in manual tests on my laptop.